llvm-dis: add page (#6049)
tldr-pages/tldr: pages/common/llvm-dis.md

# llvm-dis

> Converts LLVM bitcode files into human-readable LLVM Intermediate Representation (IR).

> > More information: <https://www.llvm.org/docs/CommandGuide/llvm-dis.html>.

Reviewer: ```suggestion
> More information: <https://manned.org/llvm-dis.1>.
```

Reviewer: Ok, wait for others opinions as well though

Reviewer 2: I agree. manned.org is a _fallback_ in case an official page is not provided. In this case, we really want to keep with the llvm.org website instead, since it's an official source.

> - Print a bitcode file as LLVM IR to stdout:

Reviewer 2: Not sure this quite makes sense. What about this?
```suggestion
- Convert a bitcode file as LLVM IR and write the result to stdout:
```

> - Output a bitcode file as LLVM IR to standard output:

Reviewer 3: `print` is a better verb for basic English speakers
```suggestion
- Print a bitcode file as LLVM IR to stdout:
```

`llvm-dis {{path/to/input.bc}} -o -`

> - Convert a bitcode file to a LLVM IR file with the same filename:

Reviewer 4: ```suggestion
- Convert a bitcode file to an LLVM IR file with the same filename:
```

> - Convert a bitcode file to an LLVM IR file with the same filename:

Reviewer 2: Does it replace the file, or create a new file with a different file extension alongside it?

Author: Creates a new file with a different extension alongside

Reviewer 2: Ah, I see. Could we reword this a bit to clarify that it doesn't overwrite the input file please?

Reviewer 5: I think it's already pretty clear, since it's obvious that a bitcode file and an LLVM IR file have different extensions.

Reviewer 2: Very well. I'm not experienced with this command, so I'll defer to your judgement here @marchersimon

> - Convert a bitcode file to LLVM IR with the same filename:

Reviewer 3: ```suggestion
- Convert a bitcode file to a LLVM IR file with the same filename:
```

`llvm-dis {{path/to/file.bc}}`

> - Convert a bitcode file to LLVM IR:

Reviewer 3: ```suggestion
- Convert a bitcode file to a LLVM IR file:
```

> - Convert a bitcode file to a LLVM IR file:

Reviewer 4: ```suggestion
- Convert a bitcode file to an LLVM IR file:
```

> - Convert a bitcode file to an LLVM IR file:

Reviewer 2: Perhaps this will read better?
```suggestion
- Convert a bitcode file to LLVM IR, writing the result to a specified file:
```

> - Convert a bitcode file to LLVM IR, writing the result to a specified file:

Reviewer 4: ```suggestion
- Convert a bitcode file to LLVM IR, writing the result to the specified file:
```
